content/rc/databases/high-availability.md

```diff
@@ -0,0 +1,29 @@
+Title: High Availability
+description:
```

Comment:
Can we add a description that indicates that this page is for Redis Enterprise cloud HA?

Reply:
Good idea.